.github/workflows/pr_build.yml

```diff
@@ -10,6 +10,9 @@ jobs:
     env:
       TAMAGO_VERSION: 1.21.3
       TAMAGO: /usr/local/tamago-go/bin/go
+      LOG_ORIGIN: TEST-LOG
+      LOG_PRIVATE_KEY: /tmp/log.sec
```

Comment:
Is there actually a file at these paths? Seems like no, in which case a comment saying that this is just to get the Makefile to pass checks would clarify this.

Reply:
Yes, it gets created at L29-31 along with the other throwaway keys.

Reply:
Ta, I see that now.